tests/Integration/Behaviour/Features/Scenario/Product/update_seo.feature

```diff
@@ -383,3 +384,29 @@ Feature: Update product SEO options from Back Office (BO)
       | redirect_target | clothes                               |
       | redirect_name   | Home > Clothes                        |
       | redirect_image  | http://myshop.com/img/c/{clothes}.jpg |
+
+  Scenario: Empty friendly-urls should be auto-filled using product name value
+    And I add product "product4" with following information:
+      | type        | standard |
+      | name[en-US] |          |
+      | name[fr-FR] |          |
+    Then product "product4" localized "name" should be:
+      | locale | value |
+      | en-US  |       |
+      | fr-FR  |       |
+    When I update product "product4" basic information with following values:
+      | name[en-US] | en product 04 |
+      | name[fr-FR] | fr product 04 |
+    Then product "product4" localized "name" should be:
+      | locale | value         |
+      | en-US  | en product 04 |
+      | fr-FR  | fr product 04 |
+    When I update product product4 SEO information with following values:
```

Comment:
```suggestion
    When I update product product4 SEO information with following values:
```

What is default link_rewrite before this update ?

Reply:
I mentioned in comment couple lines later `    # UpdateProductBasicInformationCommand also modifies link_rewrite, so it was already changed during name update
    # therefore this test is not the most accurate`
    
so im not really sure what to do with this test, its abit useless :smile: 
I could make it work as expected if I update the product name without using UpdateBasicInformationCommand (probably some legacy method), but idk if its worth it and how would i call it like? :thinking: 
wdyt ? @Progi1984 @jolelievre ?